### DISCUSSION TITLE
How can I get Student Developer Pack?

### DISCUSSION THREAD

**Discussion by dk172923** [community/community#110774] [GitHub Education]
### Hi everyone,

I'm Dhineshkumar currently in my 3rd year of Computer Science Engineering. I don't have a college issued email address. How can I get GitHub Student Developer Pack without a college issued email address? Please let me know if anyone knows how resolve my issue....

**Comment by SebastienXia** [comment]
Hi, @dk172923  and welcome to the Github Community,

The email address issued by your establishment **is not obligatory**, you can provide your **personal** email address (as I did), this will not risk anything in the process, but on the other hand to confirm your belonging to a school you will need provide a **student card** or **other documents confirming your membership in a school for this year 2023-2024**. You will see during the process :)

![Capture](https://github.com/community/community/assets/105004912/83ce20a7-e09d-428f-8d1b-d8a0ca2b79cf)
I still don't have my benefits because it takes time : https://github.com/orgs/community/discussions/58094#discussion-5300061

Sincerely  @SebastienXia
